source/docs/software/advanced-controls/introduction/introduction-to-feedforward.rst

```diff
@@ -55,7 +55,7 @@ where :math:`V` is the applied voltage, :math:`\theta` is the angular displaceme
 Using the Feedforward
 ---------------------
 
-In order to use the feedforward, we need to plug in values for each unknown in the above voltage-balance equation *other than the voltage*.  As mentioned :ref:`earlier <docs/software/advanced-controls/introduction/picking-control-strategy:Obtaining Models for Your Mechanisms>`, the values of the gains :math:`K_s` etc. can be obtained either through theoretical modeling with `ReCalc <https://www.reca.lc/>`__ or explicit measurement with :doc:`SysId </docs/software/pathplanning/system-identification/introduction>`.  That leaves us needing values for velocity, acceleration, and (in the case of the arm feedforward) position.
+In order to use the feedforward, we need to plug in values for each unknown in the above voltage-balance equation *other than the voltage*.  As mentioned :ref:`earlier <docs/software/advanced-controls/introduction/picking-control-strategy:Obtaining Models for Your Mechanisms>`, the values of the gains :math:`K_g`, :math:`K_v`, :math:`K_a`. can be obtained through theoretical modeling with `ReCalc <https://www.reca.lc/>`__. Explicit measurement with :doc:`SysId </docs/software/pathplanning/system-identification/introduction>` will yield the aforementioned gains in addition to :math:`K_s`. That leaves us needing values for velocity, acceleration, and (in the case of the arm feedforward) position.
```

Comment:
Is there a reason `K_g` is mentioned with `K_v` and `K_a` but `K_s` is mentioned only later?
Also, remove the period after `K_a`.

Reply:
Yes, K_s cannot be found with recalc so I did not include it. That's what was misleading orginally.